lib/screens/authentication/reset_password_screen.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:doctors_of_kenya/constants/constants.dart';
 import 'package:doctors_of_kenya/utilities/utilities.dart';
+import 'package:doctors_of_kenya/widgets/global_dialog.dart';

```

Comment:
@wansom There's a widgets.dart file in the widgets folder, you can export the global_dialog.dart and just import the widgets.dart here